blocks/v2-columns/v2-columns.js

```diff
@@ -42,14 +45,46 @@ export default async function decorate(block) {
     });
 
     const headings = [...col.querySelectorAll('h1, h2, h3, h4, h5, h6')];
-    headings.forEach((heading) => heading.classList.add(`${blockName}__heading`));
+    headings.forEach((heading) => heading.classList.add(`${blockName}__heading`, 'h1'));
+
+    // icons
+    [...col.querySelectorAll('.icon')].forEach((icon) => {
+      const iconParentEl = icon.parentElement;
+      if (iconParentEl.children.length === 1) {
+        iconParentEl.replaceWith(icon);
+      }
+    });
 
-    const pretitleText = headings[0]?.previousElementSibling;
+    const prevEl = headings[0]?.previousElementSibling;
+    const pretitleText = prevEl && !prevEl.classList.contains('icon') && prevEl.textContent;
 
     if (pretitleText) {
       const pretitle = createElement('span', { classes: `${blockName}__pretitle` });
-      pretitle.textContent = pretitleText.textContent;
-      pretitleText.replaceWith(pretitle);
+      pretitle.textContent = pretitleText;
+      prevEl.replaceWith(pretitle);
     }
   });
+
+  // logic for info variant
+  if (block.classList.contains(`${blockName}--info`)) {
+    const headings = [...block.querySelectorAll('h3, h4, h5, h6')];
+    const h2List = [...block.querySelectorAll('h2')];
+
+    headings.forEach((h) => {
+      h.classList.add('h5');
```

Comment:
Careful, for some reason you are adding 2 classes in the same item
<img width="427" alt="image" src="https://github.com/Netcentric/vg-macktrucks-com-rd/assets/3090999/157291d8-c716-4aca-8e2b-aec51caa61f3">